README.rdoc

```diff
@@ -22,9 +22,14 @@ Configure your environment. For example, create an initializer in Rails in <tt>c
     TaxCloud.configure do |config|
       config.api_login_id = 'your_tax_cloud_api_login_id'
       config.api_key = 'your_tax_cloud_api_key'
-      config.usps_username = 'your_usps_username'
+      config.usps_username = 'your_usps_username' # optional
+      config.open_timeout = 1 # optional
+      config.read_timeout = 1 # optional
     end
 
+The <tt>open_timeout</tt> and <tt>read_timeout</tt> options are used to specify a waiting time for the TaxCloud web service
+response. The default values are too high, and if the service is not responding you will have to wait for the response a very long time.
```

Comment:
A good idea, but actually don't know how much they should be ;) May be someone has a bad network connection and he should be able to increase it.